x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -192,7 +192,7 @@ export const AlertIndex = memo(() => {
 
   const pagination = useMemo(() => {
     return {
-      pageIndex,
+      pageIndex: pageIndex || 0,
```

Comment:
@dplumlee @peluja1012 Because of the cursor-based pagination, `pageIndex` may be undefined, but this won't happen until we actually hook up cursor-based pagination. Added this assertion for now, until we figure out what we want this to look like.